VISION.md

```diff
@@ -0,0 +1,207 @@
+# Living Documentation Generator
+
+- [Purpose of this Page](#purpose-of-this-page)
+- [Regimes and Their Usage](#regimes-and-their-usage)
+  - [DocumentedUserStories](#documenteduserstories)
+  - [DocumentedFeatures](#documentedfeatures)
+  - [DocumentedRequirements](#documentedrequirements)
+  - [Coverage matrix](#coverage-matrix)
+  - [Repositor(y/ies) YML Files](#repository-yml-files)
+  - [Incidents](#incidents)
+  - [Release Notes](#release-notes)
+  - [User Guide](#user-guide)
+- [Rationale for an Isolated GitHub Repository](#rationale-for-an-isolated-github-repository)
+- [Proposed Issue Content Examples](#proposed-issue-content-examples)
+- [Content Formatting for Documentation Types (US, Features, Req)](#content-formatting-for-documentation-types-us-features-req)
+- [Examples](#examples)
+  - [DocumentedUserStory GH Issue](#documenteduserstory-gh-issue)
+  - [DocumentedFeature GH Issue](#documentedfeature-gh-issue)
+  - [DocumentedRequirement GH Issue](#documentedrequirement-gh-issue)
+
+## Purpose of this Page
+The goals of this document are to:
+- Outline features yet to be implemented.
+- Expand upon the content provided in README.md.
+
+## Regimes and Their Usage
+Outlined below are the various regimes, their intended purposes, and the specific inputs and outputs associated with each.
```

Comment:
What `regime` stand here for? Who is using regimes? What is the purpose to have them.

Reply:
Added regime term explanation.